templates/base.html

```diff
@@ -104,7 +105,7 @@
             </div>
             <div class="row">
                 <div class="col-lg-4 col-lg-offset-2">
-                    <p>We are a voluntary non for profit organization of professional and hobbyists with a shared interest for the Django web framework.</p>
+                    <p>We are a voluntary non for profit organization of professional and hobbyists with a shared passsion for creating sites in the Django web framework.</p>
```

Comment:
To be picky about something you didn't change... I think it should be called "nonprofit" rather than "non for profit" :)